realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -68,17 +68,14 @@ public void generate() throws IOException, UnsupportedOperationException {
         imports.add("io.realm.internal.TableOrView");
         imports.add("io.realm.internal.ImplicitTransaction");
         imports.add("io.realm.internal.LinkView");
-        imports.add("io.realm.internal.android.JsonUtils");
         imports.add("java.io.IOException");
         imports.add("java.util.ArrayList");
         imports.add("java.util.Collections");
         imports.add("java.util.List");
-        imports.add("java.util.Date");
```

Comment:
I am pretty sure this is needed if you Model class contains date objects. Will have to look into the others. Unfortunately JavaWriter doesn't allow you to only import the Classes that is actually used :(